src/go/rdctl/cmd/snapshotCreate.go

```diff
@@ -16,13 +18,30 @@ import (
 )
 
 var snapshotDescription string
+var snapshotDescriptionFromFile string
```

Comment:
Since we shortened the option name, maybe we should shorten the variable name to match?

```suggestion
var snapshotDescriptionFrom string
```